File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Farinha/05.pl
#Programa que verifica se um ano é comum ou bissexto.
print "Digite um ano:\n";
$ano = <STDIN>;
if ($ano % 400 == 0)
{
	print "Ano bissexto\n";
}
elsif ($ano % 4 == 0 && $ano % 100 != 0)
{
	print "Ano bissexto\n";
}
else
{
	print "Ano comum\n";
}

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Farinha/05.rb
#Programa que verifica se um ano é comum ou bissexto.
puts "Digite um ano:"
ano = gets
if ano.to_i % 400 == 0
  puts "Ano bissexto"
elsif ano.to_i % 4 == 0 && ano.to_i % 100 != 0
  puts "Ano bissexto"
else
  puts "Ano comum"
end